File: listView/app/src/main/java/com/example/listview/MainActivity.java
package com.example.listview;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.content.ContextCompat;

import android.os.Bundle;
import android.text.Editable;
import android.text.TextWatcher;
import android.widget.EditText;
import android.widget.ListView;

public class MainActivity extends AppCompatActivity {
    ListView listview = null;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ListViewAdapter adapter;

        // Adapter 생성
        adapter = new ListViewAdapter();

        // 리스트뷰 참조 및 Adapter달기
        listview = (ListView) findViewById(R.id.listview1);
        listview.setAdapter(adapter);

        adapter.addItem(ContextCompat.getDrawable(this, R.drawable.dish),
                "오붕집", "직화낙지&보쌈 푸짐하고 맛있음");

        adapter.addItem(ContextCompat.getDrawable(this, R.drawable.dish),
                "스시현", "포장할정도로 푸짐했던 스시현");

        adapter.addItem(ContextCompat.getDrawable(this, R.drawable.dish),
                "애프터글로우", "월곡역 최고의 칵테일 맛집");

        adapter.addItem(ContextCompat.getDrawable(this, R.drawable.dish),
                "백소정", "두툼한 돈가스에 소바가 맛있는 백소정");

        adapter.addItem(ContextCompat.getDrawable(this, R.drawable.dish),
                "남매부대찌개", "월곡역 부대찌개 맛집");

        adapter.addItem(ContextCompat.getDrawable(this, R.drawable.dish),
                "청년고기장수", "월곡역 무한리필 고기집");

        EditText editTextFilter = (EditText) findViewById(R.id.editTextFilter);
        editTextFilter.addTextChangedListener(new TextWatcher() {
            @Override
            public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) {

            }

            @Override
            public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {

            }

            public void afterTextChanged(Editable edit) {
                String filterText = edit.toString();
                if (filterText.length() > 0) {
                    listview.setFilterText(filterText) ;
                } else {
                    listview.clearTextFilter();
                }
            }
        });
    }
}

File: matzip/app/src/main/java/com/example/matzip/MainActivity.kt
package com.example.matzip

import android.content.DialogInterface
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AlertDialog

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun event3(view: View) {
        val id: TextView = findViewById(R.id.id)
        val idVal: String = id.text.toString()
        val password: TextView = findViewById(R.id.password)
        val passVal: String = password.text.toString()
        val intent: Intent

        if(idVal.equals("dongduk") && passVal.equals("computer")) {
            intent = Intent(this, Select::class.java)
            startActivity(intent)
        } else {
            val builder = AlertDialog.Builder(this)

            builder.setTitle("아이디 혹은 비밀번호 오류!")
                .setMessage("아이디와 비밀번호를 에브리타임을 참고하여 다시 입력해보세요.")
                .setPositiveButton("확인",
                    DialogInterface.OnClickListener { dialog, id ->
                    })
            val alertDialog = builder.create()

            alertDialog.show()
        }

    }

    fun event4(view: View) {
        val intent = Intent(this, Welcom::class.java)
        startActivity(intent)
    }
}